### ISSUE TITLE
There's no pop-up window after connecting my headphones

### ISSUE THREAD

**Issue by xiaomengxinyo** [steam3d/MagicPods-Windows#395]
[MagicPodsServiceLog.zip](https://github.com/steam3d/MagicPods-Windows/files/13850251/MagicPodsServiceLog.zip)
My computer system is Windows 11, my headphones are fake AirPods, When the iPhone connects to my headphones it doesn't detect fake AirPods and can display all the power normally.My Settings always keeps the default settings, and I also try to change the search radius, update the bluetooth driver, and check the system files, still didn't solve the problem. The log file at the end of the last time I tried to connect my headphones, there was no pop-up window.
![屏幕截图(87)](https://github.com/steam3d/MagicPods-Windows/assets/48215946/abbdf484-4602-495b-b988-37a5ecd59ecc)
![屏幕截图(88)](https://github.com/steam3d/MagicPods-Windows/assets/48215946/5a76aa7d-e3de-496f-9c34-6dfbda856cd8)

**Comment by steam3d** [comment]
Fake AirPods does no show battery info in popup window. Fake AirPods does not provide this data.

**Comment by xiaomengxinyo** [comment]
Thank you for your reply. My AirPods are in the AirPods case and don't show pop-up windows when the lid is opened. Or that fake AirPods don't provide battery data and don't support displaying pop-ups?

**Comment by xiaomengxinyo** [comment]
![Video_20240107_121742_541](https://github.com/steam3d/MagicPods-Windows/assets/48215946/d0a97597-5c11-4fc5-b0a0-2e8427516d0f)

**Comment by steam3d** [comment]
Go to preferences settings, change the 'debug log' to 'trace'. Try to open the case with AirPods inside. After it send the updated log files here.

I will check the data from your Fake AirPods. Currently I tested Fake AirPods Pro on  Airoha chip.

**Comment by xiaomengxinyo** [comment]
I have done as you said, have you received the log files? My headphone chip is Airoha AB1562M,

**Comment by xiaomengxinyo** [comment]
[New.zip](https://github.com/steam3d/MagicPods-Windows/files/13850452/New.zip)
This is the Log files I cleared before and collected again.

**Comment by steam3d** [comment]
I remembered that some of users also has AB1562M. This chip is unsupported,.because I can't find where to buy this Fake AirPods to add support it. Because of it you do not see any popup and extended features, but battery should displays. Turn off the "show battery percentage" and restart the MagicPods. (I added legacy battery support, so some of fake airpods can display left, right battery info)

**Comment by steam3d** [comment]
I will add AB1562M when find the fake AirPods with this chip.

**Comment by xiaomengxinyo** [comment]
![屏幕截图(90)](https://github.com/steam3d/MagicPods-Windows/assets/48215946/6f15c4fa-f92e-4aff-bdb1-7e55b248e26e)
![屏幕截图(89)](https://github.com/steam3d/MagicPods-Windows/assets/48215946/0c5c1b35-6aa9-4466-b96e-6900a5f43d91)
There are two Show Battery percentage switches, the first is inside the Headphones, The second is inside General settings. I try to turn off the first one, and it doesn't Show any batteries. When I try to turn both off, it doesn't Show any batteries. Finally I found that the Battery was displayed only when the first Show Battery percentage remained open, But it only Shows one Battery, and I don't know what battery is.
This chip is a bit old, about three or four years ago, really can not buy, you may try to buy through some second-hand platform, I just found in the trial period does not support, if support I will buy, but if not support it does not matter, thank you for your patient reply, I will recommend your software to my friends.

**Comment by xiaomengxinyo** [comment]
If you can't get this headset, I'd be happy to help you. Android has an app called AndPods, which supports my headphones and displays pop-ups and accurate batteries, so maybe you can check it out.

**Comment by steam3d** [comment]
Does flycc android app support your AirPods?

**Comment by xiaomengxinyo** [comment]
Yes, flycc supports all airoha headphones including my headphones.

**Comment by steam3d** [comment]
Okay. I will let you know when I will have any news

**Comment by xiaomengxinyo** [comment]
Thanks.

**Comment by Craniniaxx** [comment]
Hi, I've recently bought the app after looking at the reviews and I've run into an issue here. Everything seems to ve working perfectly fine, but the popup animation doesn't appear. I've tried the search radius method but to no avail. The battery percentage was being shown in the app and in the tray icon but no popup. I have Airpods Pro 2 (2nd generation) and today I even Installed MagicAAP thinking that this might be the problem but now this has become even more absurd than before. It shows the Battery (which I double checked with iPhone and is accurate) for both the individual pods, and the case... But only in the app. Still no pop up. Please, if any one knows anything about it, guide me. I'll do anything at this point...
![17257731633281000000498576988031](https://github.com/user-attachments/assets/a19a5f78-a689-4da3-81fd-fe55fae9aef3)

**Comment by steam3d** [comment]
@xhych Fake AirPods does not support popup animation.

**Comment by xhych** [comment]
> @xhychFake AirPods不支持弹出动画。

Okay, sorry to bother you. I thought that since the iPhone has a pop-up earphone, the MagClipods could also have a pop-up.

**Comment by steam3d** [comment]
@xhych Fake AirPods have a strange implementation of popup animation. In 99% of cases, they always send the battery info as 100%, even if the AirPods have only 10% battery. I do not know how Apple devices avoid this. What is MagClipods?

**Comment by xhych** [comment]
> @xhych Fake AirPods have a strange implementation of popup animation. In 99% of cases, they always send the battery info as 100%, even if the AirPods have only 10% battery. I do not know how Apple devices avoid this. What is MagClipods?

Sorry, I typed the wrong word. It was 'magicpods'.(っ °Д °;)っ
